podcasts/Bookmarks/List/BookmarkRow.swift

```diff
@@ -60,15 +60,18 @@ struct BookmarkRow<Style: BookmarksStyle>: View {
                     Text($0)
                         .foregroundStyle(style.tertiaryText)
                         .font(style: .caption, weight: .semibold)
+                        .lineLimit(1)
                 }
 
                 Text(rowModel.title)
                     .foregroundStyle(style.primaryText)
                     .font(style: .subheadline, weight: .medium)
+                    .lineLimit(1)
```

Comment:
This come from the review from @david-gonzalez-a8c .
David should we just limit the title of the episode and the timestamp? but leave the bookmark title to grow to other lines?

Reply:
Oh got it! Just to add more context for @david-gonzalez-a8c: this is the only place in which we display the full title. If we make this change, long texts will be clipped, and the only way to see it is by editing the title.

Reply:
Thanks for the context, let's not truncate the name of the bookmark title then, same as in the player.

Reply:
I kind of would like for the artwork to be kept with the bookmark tital text to be scrollable 😅,  also are more sort options going to be added as I would like this page to be grouped by podcast, also it might be nice to have a hop functionality that jumps by podcast I think this page could get very unwieldy with our it 🤔 (look at now for Reddit)